src/models/collection-class.js

```diff
@@ -0,0 +1,57 @@
+//routing and crud functionality in this file.
+'use strict';
+
+class Collection {
+    constructor(name, model) {
+        this.model = model;
+        this.name = name;
+    }
```

Comment:
try:
```
 constructor(model, app ,name) {
    this.model = model;
    this.router(app, name);
```
this way you have the model and route. The 'app' part is for the route so you can do 'app.get'